### PR TITLE
DT-935 Update dynamic popup

### PR DESCRIPTION
**Pull request by hannesj**

### CODE REVIEW

File: app/component/map/origin-popup.cjsx [hunk 1]
```diff
@@ -1,93 +1,42 @@
...
-    @context.getStore('EndpointStore').addChangeListener @onEndpointChange
-    @context.getStore('PositionStore').addChangeListener @onPositionChange
-    if @context.getStore('EndpointStore').isPendingPopup()
-      @showEndpoint()
-
-  componentWillUnmount: =>
-    @context.getStore('EndpointStore').removeChangeListener @onEndpointChange
-    @context.getStore('PositionStore').removeChangeListener @onPositionChange
+    if @props.shouldOpen
+      setImmediate @display
```
Comment (samuliheljo) [inline app/component/map/origin-popup.cjsx]: Am I missing something here?
http://caniuse.com/#search=setimmediate

Comment (hannesj) [inline app/component/map/origin-popup.cjsx]: [This](https://github.com/HSLdevcom/digitransit-ui/blob/update-dynamic-popup/app/server.cjsx#L71)?

**Comment by samuliheljo** [comment]
I'm still getting
````/Users/samuli/github/digitransit-ui/app/component/map/dynamic-popup.js
  63:9   error  Unexpected dangling '_' in '_contentNode'     no-underscore-dangle
  66:7   error  Unexpected dangling '_' in '_updateLayout'    no-underscore-dangle
  67:7   error  Unexpected dangling '_' in '_updatePosition'  no-underscore-dangle
  68:7   error  Unexpected dangling '_' in '_adjustPan'       no-underscore-dangle
  75:9   error  Unexpected dangling '_' in '_contentNode'     no-underscore-dangle
  76:30  error  Unexpected dangling '_' in '_contentNode'     no-underscore-dangle
```

**Comment by samuliheljo** [comment]
Thanks. merging.

**Pull request merged**